### DISCUSSION TITLE
如何将字符串类型的CQ码转换为SoraSegment

### DISCUSSION THREAD

**Discussion by DontWantToName** [Hoshikawa-Kaguya/Sora#74] [Q&A] [answered]
如何将字符串类型的CQ码转换为SoraSegment

**Comment by Yukari316** [comment, marked as answer]
可以使用`Sora.Util.CQCodeUtil`中的`DeserializeCqCode(string str)`

对应源代码
https://github.com/DeepOceanSoft/Sora/blob/b3c760eaa3917faca5e6de559c9d99fd70cf20b7/Sora/Util/CQCodeUtil.cs#L129-L134

**Reply by DontWantToName** [reply]
好的 谢谢你 我向你敬礼啊 (^^ゞ salute！
